src/entry.js

```diff
@@ -43,23 +43,27 @@ class Entry {
       clock: new Clock(clockId, clockTime),
     }
 
-    // If signing key was passedd, sign the enrty
-    if (keystore && signKey) {
-      entry = await Entry.signEntry(keystore, entry, signKey) 
+    // If acl was passed, sign the entry (=authorize)
+    if (acl) {
```

Comment:
I think this should be a breaking change, providing an ACL should be mandatory from now on